aspnetcore/host-and-deploy/docker/visual-studio-tools-for-docker.md

```diff
@@ -46,53 +48,119 @@ If the target framework is .NET Core, the **OS** drop-down allows for the select
 The Visual Studio Tools for Docker don't support adding Docker to an existing ASP.NET Core project targeting .NET Framework. For ASP.NET Core projects targeting .NET Core, there are two options for adding Docker support via the tooling. Open the project in Visual Studio, and choose one of the following options:
```

Comment:
I suggest doing the positive (what IS supported) first :) So, "For ASP.NET Core projects..." and then bullet list that follows. Then the piece about ASP.NET Core targeting .NET Framework not being supported, as its own paragraph after the bullet list.

Reply:
You're absolutely right. It's best to lead with the positive stuff.